[cmake] remove usage of kodi-platform

olafhering opened the pull request:
The pvr.argustv binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.

ksooo commented:
@olafhering could you please look at #88 which is basically the same, but slightly different wrt debian handling. Not sure what is correct, though @Rechi

Rechi reviewed (changes requested):
This will break the debian packaging as it now misses p8-platform (was pulled in by libkodiplatform-dev).

ksooo commented:
> This will break the debian packaging as it now misses p8-platform (was pulled in by libkodiplatform-dev).

 @olafhering it seems you have to fix all the related PRs you've submitted. :-/

olafhering commented:
I will adjust my script and add p8-platform as needed. Thanks for review.

olafhering commented:
But before that, the base system has to be fixed to avoid false failures.

Rechi commented:
You may also have to add `libtinyxml-dev` (another dependency of libkodiplatform-dev) . But both are not needed for all addons, so only add them where they are needed.

olafhering commented:
> You may also have to add `libtinyxml-dev` (another dependency of libkodiplatform-dev) . But both are not needed for all addons, so only add them where they are needed.

Not required for this addon. Not sure how to undo the "Changes requested" thing.

olafhering committed:
[cmake] remove usage of kodi-platform

The pvr.argustv binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.
Add dependency to libp8-platform-dev because libkodiplatform-dev
pulled that dependency in as a side effect. Fix that bug too by
adding an explicit dependency.

Signed-off-by: Olaf Hering <olaf@aepfle.de>

olafhering commented:
Doing development outside of master makes contributions cumbersome.
Good luck.
